### ISSUE TITLE
CI action error due to inaccessibility of k8s cluster

### ISSUE THREAD

**Issue by alphaX86** [layer5io/meshery-smp-action#68]
#### Description

Meshery SMP Action is failing on some CI runs due to inaccessibility of k8s cluster present in the workflow

#### Expected Behavior

The error must be rectified and CI run should be free of errors

**Comment by gyohuangxin** [comment]
@alphaX86 Thank you very much.

**Comment by gyohuangxin** [comment]
@alphaX86  oops, the actions failed: https://github.com/layer5io/meshery-smp-action/actions/runs/3407625025/jobs/5667478085#step:3:79

**Comment by alphaX86** [comment]
From the logs it shows that it didn't have permission to extract files 😭 oh damn.... Should we try an alternative function for this? Or shall we wait for subsequent CI action runs to see results?

**Comment by gyohuangxin** [comment]
@alphaX86 I give you the maintain access to this repo, you can run a [configurable benchmark test on self-hosted runner](https://github.com/layer5io/meshery-smp-action/actions/workflows/configurable-benchmark-test-self-hosted.yaml) if it works.

**Comment by alphaX86** [comment]
👀👀 ok, will do 👍

**Comment by gyohuangxin** [comment]
@alphaX86 Do you have any updates?

**Comment by alphaX86** [comment]
I ran the CI action on the repo and checked. Seems its an error on the permissions for extracting installation files during k8s setup. IDK what exactly causes this. I referred this in net and as per the suggestions I even re-ran the action. No joy :/

Likely we might need to change the k8s action to test it. What's your opinion? @gyohuangxin

**Comment by gyohuangxin** [comment]
@alphaX86 Please go ahead, the only difference is the version of `manusa/actions-setup-minikube`, so IMO maybe you can research what's changeed in the new version.

**Comment by alphaX86** [comment]
For CIL cluster, seems it's restricted to use actions within it. Similar error is also caused in this repo I saw...
https://github.com/engineerd/setup-kind/pull/14
Seems manual method is the way for solving in case of self-hosted runners :/
